science/ecCodes/files/patch-no-jasper-deps.diff

```diff
@@ -0,0 +1,20 @@
+diff -Nurd eccodes-2.13.0-Source.orig/CMakeLists.txt eccodes-2.13.0-Source/CMakeLists.txt
+--- eccodes-2.13.0-Source.orig/CMakeLists.txt	2019-07-08 14:06:44.000000000 +0200
++++ eccodes-2.13.0-Source/CMakeLists.txt	2019-07-19 13:27:24.158310812 +0200
+@@ -196,11 +196,11 @@
+     #       which can affect future package discovery if not undone by the caller.
+     #       The current CMAKE_PREFIX_PATH is backed up as _CMAKE_PREFIX_PATH
+     #
+-    set(CMAKE_WARN_DEPRECATED OFF) # Suppress deprecation message
+-    ecbuild_add_extra_search_paths( jasper )
+-    find_package( Jasper )
+-    set(CMAKE_PREFIX_PATH ${_CMAKE_PREFIX_PATH})    # Restore CMAKE_PREFIX_PATH
+-    set(CMAKE_WARN_DEPRECATED ON)  # Remove suppression
++    # set(CMAKE_WARN_DEPRECATED OFF) # Suppress deprecation message
++    # ecbuild_add_extra_search_paths( jasper )
++    # find_package( Jasper )
++    # set(CMAKE_PREFIX_PATH ${_CMAKE_PREFIX_PATH})    # Restore CMAKE_PREFIX_PATH
++    # set(CMAKE_WARN_DEPRECATED ON)  # Remove suppression
```

Comment:
I never understand patches that comment lines out. Why not just remove the lines? It will make the patch smaller and easier to read.

Reply:
Yes, one can argue either way. I'll remove the lines, maybe that suits most people.

It looks to me that ECMWF needs looking for `jasper` for their own needs. It's not clear why. And this is no `-DUSE_JASPER=OFF` functionality, as the `CMakeLists.txt` **forces** looking for `jasper` no matter what!

So thus the only option is to take these lines, which **overrule** the deprecation checking, away. I will notify the makers, indeed.